### PR TITLE
Veoss 422 - Returns email confirmation

### PR DESCRIPTION
**Pull request by warrenskiwork1979**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -76,6 +76,13 @@ microservice {
         port     = 9514
         basePath = ""
       }
+
+      email {
+        protocol = "http"
```
Comment (darbyo) [inline conf/application.conf:81]: Do we have the PRs for app-conf-{env} we will need to over ride these values

**Review by darbyo — CHANGES_REQUESTED** [review]

**Comment by darbyo** [comment]
app/viewmodels/checkAnswers/SalesAtVatRateFromEuSummary.scala Cant comment inline on this as there are no lines in the diff. But do we need this file? Its empty...

**Comment by darbyo** [comment]
Also, are there some commits missing? We have added the emailConnector but no emailService, and neither are being called on successful submit

File: conf/application.conf (hunk 1 above)
Comment (warrenskiwork1979) [reply conf/application.conf:81]: Added in app-config-base for returns

**Review by DevAndyroo — APPROVED** [review]

**Pull request merged**